packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -8,6 +10,8 @@ import { Transition } from 'router_js';
 import EmberRouter, { QueryParam } from '../system/router';
 import { extractRouteArgs, resemblesURL, shallowEqual } from '../utils';
 
+const ROUTER = symbol('ROUTER') as string;
```

Comment:
Why do you have to cast to string here? Most of the other usages in the codebase don't seem to cast...

Reply:
It's weird I'm getting lint error if not doing so
```
packages/@ember/-internals/routing/lib/services/router.ts:69:23 - error TS2538: Type 'symbol' cannot be used as an index type.

69     let router = this[ROUTER];
```

Reply:
As to explain why it's only erroring here, I'm seeing these patterns in the codebase
<img width="665" alt="image" src="https://user-images.githubusercontent.com/8369011/93967481-29c66180-fd1c-11ea-8310-e203131250ce.png">

Reply:
Ugh. Maybe we should make `symbol` be typed this way to begin with (to avoid each callsite from having this pain). Either way, seems fine to do separately.